evaluator/src/test/kotlin/LicenseViewTest.kt

```diff
@@ -0,0 +1,210 @@
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.ort.evaluator
+
+import io.kotlintest.shouldBe
+import io.kotlintest.specs.WordSpec
+
+class LicenseViewTest : WordSpec() {
+    init {
+        "All" should {
+            "return the correct licenses" {
+                LicenseView.All.licenses(packageWithoutLicense, emptyList()) shouldBe emptyList()
+
+                LicenseView.All.licenses(packageWithoutLicense, detectedLicenses) shouldBe listOf(
+                    Pair("LicenseRef-a", LicenseSource.DETECTED),
+                    Pair("LicenseRef-b", LicenseSource.DETECTED)
+                )
+
+                LicenseView.All.licenses(packageWithOnlyConcludedLicense, emptyList()) shouldBe listOf(
+                    Pair("LicenseRef-a", LicenseSource.CONCLUDED),
+                    Pair("LicenseRef-b", LicenseSource.CONCLUDED)
+                )
+
+                LicenseView.All.licenses(packageWithOnlyConcludedLicense, detectedLicenses) shouldBe listOf(
+                    Pair("LicenseRef-a", LicenseSource.CONCLUDED),
+                    Pair("LicenseRef-b", LicenseSource.CONCLUDED),
+                    Pair("LicenseRef-a", LicenseSource.DETECTED),
+                    Pair("LicenseRef-b", LicenseSource.DETECTED)
+                )
+
+                LicenseView.All.licenses(packageWithOnlyDeclaredLicense, emptyList()) shouldBe listOf(
+                    Pair("license-a", LicenseSource.DECLARED),
+                    Pair("license-b", LicenseSource.DECLARED)
+                )
+
+                LicenseView.All.licenses(packageWithOnlyDeclaredLicense, detectedLicenses) shouldBe listOf(
+                    Pair("license-a", LicenseSource.DECLARED),
+                    Pair("license-b", LicenseSource.DECLARED),
+                    Pair("LicenseRef-a", LicenseSource.DETECTED),
+                    Pair("LicenseRef-b", LicenseSource.DETECTED)
+                )
+
+                LicenseView.All.licenses(packageWithConcludedAndDeclaredLicense, emptyList()) shouldBe listOf(
+                    Pair("LicenseRef-a", LicenseSource.CONCLUDED),
+                    Pair("LicenseRef-b", LicenseSource.CONCLUDED),
+                    Pair("license-a", LicenseSource.DECLARED),
+                    Pair("license-b", LicenseSource.DECLARED)
+                )
+
+                LicenseView.All.licenses(packageWithConcludedAndDeclaredLicense, detectedLicenses) shouldBe listOf(
+                    Pair("LicenseRef-a", LicenseSource.CONCLUDED),
+                    Pair("LicenseRef-b", LicenseSource.CONCLUDED),
+                    Pair("license-a", LicenseSource.DECLARED),
+                    Pair("license-b", LicenseSource.DECLARED),
+                    Pair("LicenseRef-a", LicenseSource.DETECTED),
+                    Pair("LicenseRef-b", LicenseSource.DETECTED)
+                )
+            }
+        }
+
+        "ConcludedOrRest" should {
+            "return the correct licenses" {
```

Comment:
Would it make sense to also split up those to have test names which are telling what the test is about?